src/transformers/models/led/modeling_tf_led.py

```diff
@@ -2236,24 +2236,24 @@ def get_decoder(self):
     )
     def call(
         self,
-        input_ids=None,
-        attention_mask=None,
-        decoder_input_ids=None,
-        decoder_attention_mask=None,
-        head_mask=None,
-        decoder_head_mask=None,
-        encoder_outputs: Optional[Union[Tuple, TFLEDEncoderBaseModelOutput]] = None,
```

Comment:
I've replaced this to the `|` format for consistency. [There are other places](https://github.com/huggingface/transformers/pull/25853/files#diff-8408f21e0a00169697af1da6fe22472fda43d7e5666eefe81fe55fe417a672acR1193) where there's a mix of approaches that I didn't fix as all the type hints were present. Let me know if you want me to give them some consistency

Reply:
I think the | format is preferred! The main reason is that it looks nicer in the automatically generated documentation

Reply:
Ahh, great, I've then [added a commit](https://github.com/huggingface/transformers/pull/25853/commits/b42847326f49a8e61ab822b5a53f8a5720c4595e) to make them consistent within the method